photutils/psf/tests/test_photometry.py

```diff
@@ -152,6 +151,9 @@ def test_psf_photometry_niters(sigma_psf, sources):
         assert_array_equal(cp_image, image)
 
 
+@pytest.mark.filterwarnings('ignore:Both init_guesses and finder are '
```

Comment:
Blanket ignore here because these warnings are not consistent for all the cases that pass through `parametrize`.